lib/v2/screens/transfer/send/send_confirmation/send_confirmation_screen.dart

```diff
@@ -34,7 +34,7 @@ class SendConfirmationScreen extends StatelessWidget {
             leading: IconButton(
           icon: const Icon(Icons.close),
           onPressed: () {
-            Navigator.of(context).pop();
+            Navigator.of(context).popUntil((route) => route.isFirst);
```

Comment:
This change is made to take the user to the page they came from instead of back to the scanner.